src/open_company_web/components/topic_overlay_edit.cljs

```diff
@@ -210,7 +211,7 @@
     {:data fixed-growth-data}))
 
 (defn data-to-save [owner topic]
- (let [topic-kw (keyword topic)
+  (let [topic-kw (keyword topic)
```

Comment:
Indenting here but not indenting the following let-exprs doesn't seem consistent.